src/source/PeerConnection/PeerConnection.h

```diff
@@ -99,6 +103,15 @@ typedef struct {
     UINT16 MTU;
 
     NullableBool canTrickleIce;
+
+    // congestion control
+    // https://tools.ietf.org/html/draft-holmer-rmcat-transport-wide-cc-extensions-01
+    UINT16 twccExtId;
+    volatile SIZE_T transportWideSequenceNumber;
```

Comment:
top of the struct is `RtcPeerConnection peerConnection` which i guess should be there for api/abi compatibility
what would you recommend?

Reply:
Oh, yeah. The RtcPeerConnection is the public facing struct which only has a single field - version that's UINT32. The internal structure wraps the public struct. 

We can't simply add it right after this structure as in case of 64bit architecture and tight packing we will have an issue.

I would recommend the following layout:

```
typedef struct {
    RtcPeerConnection peerConnection;
    UINT32 padding;
    volatile SIZE_T transportWideSequenceNumber;
...

```

Thoughts?

Reply:
ok

Reply:
i hope RtcPeerConnection never changes :) or else it will screw things up

Reply:
Indeed, we should be careful with structures. This is one of the reasons why most of the monikers we use that are under our control are nothing but handles which are 64 bit. We then do the internal mapping or casting to the internal structure.